src/loader.js

```diff
@@ -48,6 +48,17 @@ class Loader {
     return Loader.LOADER_METHODS[extname](filepath, options);
   }
 
+  static loadPattern(filepath) {
+    return Loader.loadFile(filepath)
+      .then(pattern => {
+        return JSON.parse(pattern);
+      })
+      .catch(err => {
```

Comment:
Our coding standards prefers unused variables not to be around:

```suggestion
      .catch(() => {
```